source/tutorials/apache-spark.rst

```diff
@@ -4,35 +4,43 @@ Apache\* Spark\*
 ################
 
 This tutorial describes how to install, configure, and run Apache Spark on
-|CL-ATTR|. Apache Spark is a fast general-purpose cluster computing system with
+|CL-ATTR| on a single machine running the master daemon and a worker daemon.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+Apache Spark is a fast, general-purpose cluster computing system with
 the following features:
 
 *  Provides high-level APIs in Java\*, Scala\*, Python\*, and R\*.
 *  Includes an optimized engine that supports general execution graphs.
 *  Supports high-level tools including Spark SQL, MLlib, GraphX, and Spark
    Streaming.
 
-In this tutorial, you will install Spark on a single machine running the
-master daemon and a worker daemon.
 
 Prerequisites
 *************
 
-This tutorial assumes you have installed |CL| on your host system.
-For detailed instructions on installing |CL| on a bare metal system, visit
-the :ref:`bare metal installation guide <bare-metal-install-desktop>`.
+* |CL| installed on your host system.
 
-Before you install any new packages, update |CL| with the following command:
+  For detailed instructions on installing |CL| on a bare metal system, visit
+  the :ref:`bare metal installation guide <bare-metal-install-desktop>`.
 
-.. code-block:: bash
+* Before installing any new packages, update |CL| with the following command:
 
-   sudo swupd update
+   .. code-block:: bash
```

Comment:
Just one minor formatting change. 
Please left-align `.. code-block::` with B in Before, and assure sudo is left-aligned with c in `code-block::`. 
It's fussy stuff. It's python...🐐 ⛰

Reply:
Done! Thanks for explaining the error.